scripts/dump_bin.py

```diff
@@ -45,6 +47,7 @@ def __init__(
         exclude_fields: str = "",
         include_fields: str = "",
         limit_nums: int = None,
+        is_convert_category: bool = False,
```

Comment:
Why do we have to use this parameter to control whether to convert the string to index? 

Will it be simpler if we only convert it before dumping it to `.bin` files?